README.md

```diff
@@ -1,2 +1,85 @@
+   2. Select the metric you want. For example, CPU utilization for staging backend.
+   3. Click on `Select metric` button.
+   4. In the metric options, set `Statistic` to "Maximum"
+   5. Set period to `1 minute`.
+   6. Leave the other options like metric name, service name etc as default.
+4. In the `Conditions` section:
+   1. Choose threshold type to be `Static`
+   2. When CPU utilization is `Greater` or `Greater/Equal`
+   3. Threshold value to 70 (or whatever you want).
+   4. Click on `Next` button
+5. In the `Notification` section:
+   1. `Alarm state trigger` to be "Alert"
+   2. Select an existing SNS topic
+   3. Inside the send notification to, select the SNS topic `plio-staging-ecs-alert` you created in previous step.
+   4. Click on `Next` button
+6. Set the alarm name. For example: `plio-staging-ecs-backend-cpu-70`
+7. Add alarm description.
+8. Finally, click on `Create alarm` button after reviewing your configurations.
+
+#### Note for auto-scaling
+For production ECS, you may already see existing alarms if you have enabled auto-scaling.
+- You may wish to create new alarms by following the steps mentioned above or can update the existing ones that were automatically created.
+- If you're editing them, you can simply go to Notification section and add the SNS topic.
+- **Note:** PLEASE MAKE SURE YOU DO NOT UPDATE ANYTHING ELSE AS THIS MAY IMPACT THE AUTO-SCALING FUNCTIONALITY.
+
+### 3. Set up Discord Webhook
+1. Create a new Channel on your Discord. Name it as `plio-logs`.
+2. Go to `Channel settings > Integrations > Webhooks`
+3. Create a new webhook.
+4. Name the webhook as `AWS`.
+5. Set the webhook channel to `plio-logs`.
+6. Copy the webhook URL for later use.
+
+### 4. Set up AWS Lambda function
+1. Go to your Lambda dashboard and click on `Create function` button.
+2. Name the lambda function. For example: `plio-staging-discord-alerts`.
+4. Choose `Runtime` to "Python 3.8"
+5. In the execution role section, select `Create a new role with basic Lambda permissions` option.
+6. Click on `Create function` button at the bottom. You will see the function in your lambda function list.
+5. Go to the `Code` tab. You will see a file named `lambda_function.py`.
+6. Replace all the code inside that file with our [lambda_function.py](./lambda_function.py) file.
+7. Deploy the code changes.
+8. Go to `Configuration` tab and switch to `Environment variables`.
+9. Add a new environment variable:
+   1. Key should be `WEBHOOK_URL`
+   2. In value, paste the webhook url you copied from Discord.
+10. Next, go to `Triggers` page within the `Configuration` tab
+   3. Click on `Add Trigger`.
+   4. `Select trigger` to be "SNS".
+   5. `SNS topic` to be "plio-staging-ecs-alert" that you had created above.
+   6. Click on `Add`.
+   7. After adding this, if you go to SNS dashboard and see open the SNS topic, you will see an entry in the subscribers list for the AWS Lambda and marked as confirmed subscription.
+
+
+## Testing
```

Comment:
this whole thing needs to be set up for production right?

Reply:
I have set up new alarms and alerts for staging. So staging is covered.

I saw the auto-scaling alarms were already configured for production. So I went ahead and added the notification in there too. That means production should be covered too. We can do some load testing to see if we're getting alerts.